doc/content/lfr/heterogeneous_single_assembly_3D/Griffin_standalone_LFR.md

```diff
@@ -0,0 +1,277 @@
+
+`Richardson_rel_tol` or `Richardson_abs_tol` is the tolerance used to check the convergence of the Richardson iteration. If `richardson_postprocessor` is specified, its `PostProcessor` value is used as the convergence metric. Otherwise Griffin uses the L2 norm difference of the angular flux solution between successive Richardson iterations, which is added to `richardson_postprocessor` with the name of `flux_error` internally. `Richardson_rel_tol` is the minimum value for the ratio of the `richardson_postprocessor` value of the current iteration to that of the first iteration. `richardson_value` is for the console output purpose to show the history of `PostProcessor` values over Richardson iterations. `inner_solve_type` is about the way to perform the inner solve of the Richardson iteration. There are three options: `none`, `SI` and `GMRes`. `none` is just a direct transport operator inversion per residual evaluation, while scattering source is updated together for `SI` and `GMRes` per residual evaluation. The latter two options involve more number of transport sweeps per residual evaluation than `none` does, leading to the reduction of the number of residual evaluations and possibly the total run time. For `GMRes`, the scattering source effect is accounted for at once by performing the GMRes iterations and `max_inner_its` is the maximum number of GMRes iterations. For `SI`, the scattering source effect is accounted for by performing source iterations and `max_inner_its` is the maximum number of source iterations. The whole energy group is solved at once for `enable_group_sweep=false` (default) and a within-group linear system is solved group-by-group for `enable_group_sweep=true` with the scattering source being updated during the group sweep using the latest flux solution. `max_thermal_its` (the number of thermal iterations) is not relevant to this problem since there is no up-scattering for the fast reactor application.
+
+!listing lfr/heterogeneous_single_assembly_3D/neutronics_standalone/neutronics.i start=[Executioner] end=[] include-end=True
+
+### Flux normalization for producing the total power
+
+The flux solution can be normalized for producing the total power using the `[PowerDensity]` block. `power` is the user-specified total power and `power_density_variable` is a required `AuxVariable` name for power density. Power is calculated using the `kappaFission` cross section in the cross section library which is the total recoverable energy per fission multiplied by the fission cross section and does not account for non-fission heating.
```

Comment:
Why not account for non-fission heating? We have a control parameter `legacy_energy_deposition_definition`. Oh, looks like `[PowerDensity]` block does not have this.

Reply:
Suggest to replace this with a `!alert note` and add that this will be added in the future.

Reply:
`!alert note` is added.